addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/handler/OnkyoHandler.java

```diff
@@ -0,0 +1,344 @@
+                break;
+            case CHANNEL_VOLUME:
+                if (command instanceof PercentType) {
+                    sendCommand(EiscpCommandRef.VOLUME_SET, command);
+                } else if (command.equals(IncreaseDecreaseType.INCREASE)) {
+                    sendCommand(EiscpCommandRef.VOLUME_UP);
+                } else if (command.equals(IncreaseDecreaseType.DECREASE)) {
+                    sendCommand(EiscpCommandRef.VOLUME_DOWN);
+                } else if (command.equals(OnOffType.OFF)) {
+                    sendCommand(EiscpCommandRef.MUTE);
+                } else if (command.equals(OnOffType.ON)) {
+                    sendCommand(EiscpCommandRef.UNMUTE);
+                }
+                break;
+            case CHANNEL_INPUT:
+                if (command instanceof DecimalType) {
+                    selectInput(((DecimalType) command).intValue());
+                }
+                break;
+            case CHANNEL_NEXT:
+                if (command.equals(OnOffType.ON)) {
+                    if (!isPlayingNetUsb()) {
+                        playNetUsb();
+                    }
+                    sendCommand(EiscpCommandRef.NETUSB_OP_TRACKUP);
+                }
+                break;
+            case CHANNEL_PREV:
+                if (command.equals(OnOffType.ON)) {
+                    if (!isPlayingNetUsb()) {
+                        playNetUsb();
+                    }
+                    sendCommand(EiscpCommandRef.NETUSB_OP_TRACKDWN);
+                }
+                break;
+            case CHANNEL_CONTROL:
+                if (!isPlayingNetUsb()) {
+                    playNetUsb();
+                }
+                if (command instanceof PlayPauseType) {
+                    if (command.equals(PlayPauseType.PLAY)) {
+                        sendCommand(EiscpCommandRef.NETUSB_OP_PLAY);
+                    } else if (command.equals(PlayPauseType.PAUSE)) {
+                        sendCommand(EiscpCommandRef.NETUSB_OP_PAUSE);
+                    }
+                }
+                if (command instanceof NextPreviousType) {
+                    if (command.equals(NextPreviousType.NEXT)) {
+                        sendCommand(EiscpCommandRef.NETUSB_OP_TRACKUP);
+                    } else if (command.equals(NextPreviousType.PREVIOUS)) {
+                        sendCommand(EiscpCommandRef.NETUSB_OP_TRACKDWN);
+                    }
+                }
+                if (command instanceof RewindFastforwardType) {
+                    if (command.equals(RewindFastforwardType.REWIND)) {
+                        sendCommand(EiscpCommandRef.NETUSB_OP_REW);
+                    } else if (command.equals(RewindFastforwardType.FASTFORWARD)) {
+                        sendCommand(EiscpCommandRef.NETUSB_OP_FF);
+                    }
+                }
+                break;
+            default:
+                logger.debug("Received unknown channel {}", channelUID.getIdWithoutGroup());
```

Comment:
You probably mean "unknown command".
Note that you should also support the REFRESH command.